deep_qa/layers/adaptive_recurrence.py

```diff
@@ -31,7 +31,8 @@ class AdaptiveRecurrence:
     can create one copy of the weights and although the number of iterations of the memory step is
     non-deterministic, the computational graph can be statically defined - we just loop over a part of it.
     '''
-    def __init__(self, memory_network, params: "Params"):
+    # pylint: disable=dangerous-default-value, unused-argument
```

Comment:
Just saw your comment on the PR, too - if you have to have `**kwargs` here to make the API work out with different layer classes, that's fine, and you can keep the `unused-argument` pylint thing.